VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -156,39 +183,72 @@ export class ParticipantsPanelComponent implements OnInit, AfterViewInit, OnDest
     }
 
     toggleMuteAll() {
-        this.videoCallService.muteAllParticipants(!this.isMuteAll);
+        this.logger.debug(`${this.loggerPrefix} Judge is attempting to toggle mute all status`, {
+            conference: this.conferenceId,
+            current: this.isMuteAll,
+            new: !this.isMuteAll
+        });
+        this.videoCallService.muteAllParticipants(!this.isMuteAll, this.conferenceId);
```

Comment:
muting self has a log after the event, should there be one here?

Reply:
The callback (see line 105) will capture when this is successful.